app/components/post_draft/draft_handler/draft_handler.tsx

```diff
@@ -122,18 +129,23 @@ export default function DraftHandler(props: Props) {
     }, [files]);
 
     return (
-        <SendHandler
-            testID={testID}
-            channelId={channelId}
-            rootId={rootId}
-
-            // From draft handler
-            value={currentValue}
-            files={files || emptyFileList}
-            clearDraft={clearDraft}
-            updateValue={setCurrentValue}
-            addFiles={addFiles}
-            uploadFileError={uploadError}
-        />
+        <>
```

Comment:
@enahum The `<></>` seems unneeded here. Is it? Or am I missing something?

Reply:
yeah not needed, can be removed